.github/workflows/build.yml

```diff
@@ -14,27 +14,35 @@ permissions:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
-    concurrency:
-      group: ${{ github.workflow }}-${{ matrix.python }}-${{ github.ref }}
-      cancel-in-progress: true
     strategy:
       fail-fast: false
       matrix:
+        os: ["ubuntu-22.04", "ubuntu-24.04"]
         python: [
           "3.7",
           "3.8",
```

Comment:
@keszybz You mentioned we could drop them. I think it would make sense, since they are both EoL and no longer get security updates. I don't think we need to care and it would remove a lot of special casing for these particularly old versions.